File: data/data.code.csharp.client/ReqTestJs.cs
using System.Collections;
using System.Collections.Generic;


public class ReqTestJs
{
	private ulong _u64;
	private long _i64;


	public Packet Encode()
	{
		Packet packet = new Packet();
		packet.WriteUlong(this._u64);
		packet.WriteLong(this._i64);
		packet.Encode(Msg.P_REQ_TEST_JS);
		return packet;
	}


	public ulong u64
	{
		get { return this._u64; }
		set { this._u64 = value; }
	}

	public long i64
	{
		get { return this._i64; }
		set { this._i64 = value; }
	}

}

File: data/typescript.cc/ReqTestJs.ts
import Packet from '@mi/mod/Packet'


export default class ReqTestJs
{
	private _u64: Long;
	private _i64: Long;


	public Encode(): Packet {
		let packet: Packet = new Packet();
		packet.WriteUlong(this._u64);
		packet.WriteLong(this._i64);
		packet.Encode(40080);
		return packet;
	}


	public get u64(): Long { return this._u64; }
	public set u64(value: Long) { this._u64 = value; }
	public get i64(): Long { return this._i64; }
	public set i64(value: Long) { this._i64 = value; }
}
